src/sentry/static/sentry/app/styles/animations.jsx

```diff
@@ -4,9 +4,11 @@ import theme from 'app/utils/theme';
 
 export const growIn = keyframes`
   0% {
-    transform: scale(0);
+    opacity: 0;
+    transform: scale(0.75);
```

Comment:
Will this impact other usages in a detrimental way?

Reply:
it is only used in one other place (radio buttons) and they look great! this is a better way to do this animation—it has the same feeling but looks a lot smoother and less intense.